### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by best-lucky1030**
add myteams page and merge into master

### CODE REVIEW

File: src/router.js [hunk 1]
```diff
@@ -15,6 +15,11 @@ export default new Router({
       path: '/live',
       name: 'live',
       component: () => import('./views/Live.vue')
-    }
+    },
+    {
+      path: '/myteams',
+      name: 'MyTeams',
```
Comment (black-wyvern-dev) [inline src/router.js:21]: Please don't capitalize here. All code styles should be the same format.

File: src/store.js [hunk 2]
```diff
@@ -25,7 +25,8 @@ export default new Vuex.Store({
           url: 'https://api.football-data.org/v2/competitions',
         })
         .then(response => (this.competitionList = response.data.competitions.filter((item) => {
-            return item.plan == 'TIER_ONE'
+            // return item.plan == 'TIER_ONE'
```
Comment (black-wyvern-dev) [inline src/store.js:28]: Seems code prettier need. Such as space and margin of line.

**Review by black-wyvern-dev — COMMENTED** [review]
Totally, seems good. Just need a few prettier ones.
Thanks.

File: src/router.js (hunk 1 above)
Comment (best-lucky1030) [reply src/router.js:21]: thanks for your suggestion.

**Pull request merged**